scripts/model/riken.js

```diff
@@ -55,10 +55,10 @@
           var newTag = xhr.getResponseHeader('eTag');
           if (newTag !== localStorage['eTag' + name]){
             localStorage.rikenpublications ='';
-            console.log('getting new data!');
+            // console.log('getting new data!');
             RikenP.fetchAll(url, name, nextFunction); // recursive call
           } //end of if
-          console.log('got your data right here');
+          // console.log('got your data right here');
           var retreivedData =  JSON.parse(localStorage.getItem(name, data));
           RikenP.loadIntoObjectArray(eval(retreivedData));
           nextFunction();
```

Comment:
Just a thought on the Riken App HTML - 

For the form/date input, the following may help reduce user input errors/help validate data  ``<input type="date" name="bday">``.

If you have a list of well-known journals/often inputted journals, you could do an a dropdown for the journals. If they click "other journal" you could maybe open up another input field that is a text field? Just throwing out thoughts. 

Also, is the PMID a number? or text? If it is a number, you could write that as a required number in HTML5 - something like this:  `<input type="number" name="quantity" min="1" max="5">`. 

I really like the "add author" button. Very, very cool. Also, your email feature is on point. Really nice so far.